.github/actions/minty/post.js

```diff
@@ -0,0 +1,18 @@
+const core = require('@actions/github');
+const github = require('@actions/github');
```

Comment:
Same import?

Reply:
`main.js` and `post.js` are loaded at separate times. They both need imports for the things they need.

Reply:
No I mean you are importing the same thing twice. I think you want `const core = require('@actions/core')` instead.

Reply:
oh geez, yeah, you are right